src/components/dialogs/dialogAlert/dialogAlertRoot/dialogAlertRoot.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { type ReactNode } from 'react';
+import { DialogAlertRoot, type IDialogAlertRootProps } from './dialogAlertRoot';
+
+describe('<DialogAlert.Root/> component', () => {
+    const createTestComponent = (rootProps?: Partial<IDialogAlertRootProps>, children?: ReactNode) => {
```

Comment:
The root component already accept the `children` property, right? We can pass it as props
```suggestion
    const createTestComponent = (props?: Partial<IDialogAlertRootProps>) => {
```